### DISCUSSION TITLE
Raylib program running on background on RPi4 Linux

### DISCUSSION THREAD

**Discussion by reinhardkrakow** [raysan5/raylib#2450] [Q&A]
Hi everybody,

I am wandering if it is possible to run a Raylib application on background.
I add the "LogCustom" function in my program to avoid the log message on the console.
If I start my program like **./myDemo/chrono &**, Linux respond with **[1] 72043** this is the process PID and the Linux prompt is displayed.
But my program dit not display my graphic : there is nothing on the display. With the command **ps -e** I can see that my program is running.

Any idea?

BR
Reinhard
[ChronoProcess.docx](https://github.com/raysan5/raylib/files/8581642/ChronoProcess.docx)

**Comment by archie2x** [comment]
It's stuck configuring the keyboard for input in "InitKeyboard()"

You can redirect /dev/null to your stdin and that will unblock it, e.g.

```
build/examples> shapes_logo_raylib </dev/null &
```

Interestingly, if you do this, predictably you can't hit 'ESC' to quit the program anymore, but it will now exit with CTRL-C.
